### PR TITLE
[Clientside Routing] Wire in Auctions app

### PR DESCRIPTION
**Pull request by damassi**
Force PR:  https://github.com/artsy/force/pull/5064  (see for testing instructions) 

This wires the Auctions app into new App shell. Haven't checked the full flow yet since I'm not aware of every possible permutation, but things are mostly the same as this is already an SSR router app; just mounted the route config into a parent. Clicking around, registering, deleting a credit card and re-adding, placing a bid, etc, all seem to work as it should. 

#### Todo:
- [x] QA with someone on Auctions team (with @erikdstock) 

#### Summary of changes:

1) Bundle split apps at router 
1) Moved the minimal nav bar directly into the app instead of relying on server-side template
1) If `EXPERIMENTAL_APP_SHELL = true`, instead of using `location.assign` to redirect, we use `router.push(href)`. This only relates to `artwork page`<->`bid|register|etc` flows, not links directly to the auction page as it is outside of the router context.

### CODE REVIEW

**Comment by damassi** [comment]
@dblandin - reassigning to you as I see that Yuki is OOO.

**Comment by damassi** [comment]
(Reassigning to @erikdstock as @dblandin is no longer on Auctions team.)

File: src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx [hunk 1]
```diff
@@ -40,9 +43,17 @@ export class ArtworkSidebarBidAction extends React.Component<
     this.setState({ selectedMaxBidCents: newVal })
   }
 
+  // FIXME: Wire up router link here
```
[the remark this answers is not in the document]
Comment (damassi) [reply src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx]: Ah, this is fixed. See a few lines below, where we're calling `router.push` -- versus doing a hard refresh via `window.location.href`. This keeps things on the client. I'll remove the comment.

File: src/Apps/Auction/Components/AuctionFAQ.tsx [hunk 2]
```diff
@@ -150,6 +150,7 @@ export const AuctionFAQFragmentContainer = createFragmentContainer(AuctionFAQ, {
   `,
 })
 
+// FIXME: Why is this a QueryRenderer
```
[the remark this answers is not in the document]
Comment (damassi) [reply src/Apps/Auction/Components/AuctionFAQ.tsx]: For some reason we were using a query renderer to render this page, which means that it didn't SSR render. By returning the fragment to the router we get the benefits of SEO. I'll remove this code block as its no longer used.

**Comment by artsyit** [comment]
:rocket: PR was released in `v25.11.0` :rocket:

**Pull request merged**